README.md

```diff
@@ -76,3 +76,23 @@ npm run storybook
 # and in another terminal
 npm run dev
 ```
+
```

Comment:
@rowanc1 This is very strange to me. The content of the `Working with themes` section does not show up in the README although this PR is merged.

Reply:
So strange. Will sort it out tomorrow!

Reply:
It looked like a bad squash or something, I have revived the text now in the README, thanks for catching that!